src/status_im/ui/screens/wallet/components/views.cljs

```diff
@@ -3,27 +3,45 @@
   (:require [status-im.components.react :as react]
             [status-im.ui.screens.wallet.components.styles :as styles]
             [status-im.i18n :as i18n]
-            [reagent.core :as reagent]))
-
-(defn amount-input [& [{:keys [input-options style]}]]
-  (let [active? (reagent/atom false)
-        {:keys [on-focus on-blur]} input-options]
-    (fn []
-      [react/view {:flex 1}
-       [react/text {:style styles/label} (i18n/label :t/amount)]
-       [react/view (merge (styles/amount-container @active?) style)
-        [react/text-input
-         (merge
-           {:keyboard-type          :numeric
-            :placeholder            "0.000"
-            :placeholder-text-color "#ffffff66"
-            :selection-color        :white
-            :style                  styles/text-input
-            :on-focus               #(do (reset! active? true)
-                                         (when on-focus (on-focus)))
-            :on-blur                #(do (reset! active? false)
-                                         (when on-blur (on-blur)))}
-           (dissoc input-options :on-focus :on-blur))]]])))
+            [reagent.core :as reagent]
+            [status-im.components.icons.vector-icons :as vi]
+            [status-im.components.animation :as animation]
+            [status-im.utils.platform :as platform]
+            [status-im.ui.screens.wallet.components.animations :as animations]))
+
+(views/defview tooltip [label & [style]]
+  (views/letsubs [bottom-value (animation/create-value 16)
+                  opacity-value (animation/create-value 0)]
+    {:component-did-mount (animations/animate-tooltip bottom-value opacity-value)}
+    [react/view styles/tooltip-container
+     [react/animated-view {:style (merge (styles/tooltip-animated bottom-value opacity-value) style)}
+      [react/view styles/tooltip-text-container
+       [react/text {:style styles/tooltip-text} label]]
+      [vi/icon :icons/tooltip-triangle {:color :white :style styles/tooltip-triangle}]]]))
+
+(defn amount-input []
+  (let [active? (reagent/atom false)]
+    (fn [& [{:keys [input-options style error]}]]
+      (let [{:keys [on-focus on-blur]} input-options]
+        [react/view {:flex 1}
```

Comment:
for instance this is equivalent to styles/flex

Reply:
just missed this